aws-sqs-http-connector/README.md

```diff
@@ -11,7 +11,7 @@ The job of the connector is to read messages from the queue, call an HTTP endpoi
 - `SOURCE_NAME`: Optional. Name of the Source. Default is "KEDAConnector".
 - `MAX_RETRIES`: Maximum number of times an http endpoint will be retried upon failure.
 - `CONTENT_TYPE`: Content type used while creating post request.
-- `AWS_SQS_URL`: AWS SQS full URL with account id, for example http://localhost:4576/000000000000/.  
+- `QUEUE_URL`: AWS SQS full URL with account id, for example http://localhost:4576/000000000000/.  
```

Comment:
Can we use a SQS Queue URL instead of localhost (Which I think is from localstack) - it can be confusing!

Reply:
This param is used by KEDA, if change this it would cause problem with our fission mqtrigger

Reply:
So when  we create mqt trigger of kind keda via fission-cli, we pass `--metadata queueURL=value` which is then simply converted to environment variable format and plugged in deployment manifest. So changing this would break our field contract of passing metadata field used in scaledObject as given in KEDA docs